src/CalcViewModel/Common/NavCategory.cpp

```diff
@@ -60,7 +60,7 @@ bool IsGraphingModeAvailable()
 }
 
 Box<bool> ^ _isGraphingModeEnabledCached = nullptr;
-bool IsGraphingModeEnabled()
+bool IsGraphingModeEnabled(User ^ firstUser = nullptr)
```

Comment:
nit: `currentUser` would be a better name for this variable

Reply:
Resolved